### ISSUE TITLE
Validation accuracy varies widely

### ISSUE THREAD

**Issue by nahidalam** [RasaHQ/rasa_nlu#2221]
**Rasa Core version**: 0.9.0a6

**Python version**: 3.6

**Operating system** (windows, osx, ...): OSx

**Issue**:
While training the dialogue management model, I have noticed the validation accuracy varies widely. Lost likely the model is overfitting as I have a really small amount of data. I am in the early stage of development so more data will come later. Any suggestion on improving this without adding more data?

**Content of domain file** (if used & relevant):
```
slots:
  product:
    type: text
  router:
    type: text


intents:
 - greet
 - goodbye
 - order_product


entities:
 - product
 - router

templates:
  utter_greet:
    - 'Hello! My name is Tom. How can I help you today?'
  utter_goodbye:
    - 'Talk to you later.'
    - 'Bye'
  utter_affirm:
    - 'Thats right'
    - 'You are correct'
  utter_ask_product_name:
    - 'Which product?'
  utter_default:
    - "How can I help you today?"


actions:
 - utter_greet
 - utter_goodbye
 - utter_affirm
 - utter_default 
 - utter_ask_product_name
 - actions.ActionOrderProduct

```
